suites/squid/rbd/tier-2_rbd_regression.yaml

```diff
@@ -189,6 +189,31 @@ tests:
       name: Test to verify image resize operation while changing image feature
       polarion-id: CEPH-9861
 
+  - test:
+      desc: Verify RBD image with zero size
+      config:
+        resize_to: 0
+        rep_pool_config:
+          num_pools: 1
+          num_images: 2
```

Comment:
since it's basic test please keep `num_images` also 1

Reply:
As per steps mentioned in polarian: num_images is kept as 2 as we need to have 2 images, one created with size zero and another created with size 1024 later resized to zero.